### ISSUE TITLE
Lynx support (as for now FAILING)

### ISSUE THREAD

**Issue by shanduur** [pdonadeo/rofi-web-search#1]
I've forked your script to add Lynx Support, but unfortunately I am unable to get it working. You can see what I've done on my fork, [Lynx branch](https://github.com/Sheerley/rofi-web-search/tree/lynx). I was able to open Lynx, but with no luck to open it on desired url.

**Comment by gsornsen** [comment]
@Sheerley - I have an [open PR](https://github.com/pdonadeo/rofi-web-search/pull/2) for `Chromium` and `Brave` browser support.

I'd be happy to tackle this issue if/after my PR gets approved.

**Comment by pdonadeo** [comment]
@Sheerley please have a look to my lynx branch: please ignore for now that I'm using `gnome-terminal` and let me know if it works for you (ignoring the terminal).

**Comment by pdonadeo** [comment]
Thanks @gsornsen I just merged your PR. Please have a look to my [lynx branch](https://github.com/pdonadeo/rofi-web-search/tree/lynx).

**Comment by gsornsen** [comment]
Thanks @pdonadeo - starting on it now. I'm considering adding a `TERMINAL` variable under configuration, so we can handle `lynx` with multiple terminals.

Would you prefer me to initiate a PR into the `lynx` branch or the `master` branch if I get it working?

**Comment by pdonadeo** [comment]
@gsornsen ok for the new variable but please use a list like I did in the lynx branch because different terminals use different command line conventions to run a program. The vast majority use `-e` but gnome-terminal seems to prefer `--`.

**Comment by gsornsen** [comment]
@Sheerley @pdonadeo - PR for lynx support [Here](https://github.com/pdonadeo/rofi-web-search/pull/3/)

**Comment by gsornsen** [comment]
I opted to not use `CONFIG['BROWSER_PATH']` and did not add `CONFIG['TERMINAL_PATH']` for `lynx` `st` or `gnome-terminal` as subprocess should be able to determine the path of the binaries on its own since the string matches the binary name in system space. It's also possible the installation locations may change if someone compiled the binary themselves, such as I did with my custom fork of `st`

Example:
```bash
(base) gsornsen@pop-os:~/git/rofi-web-search$ which lynx
/usr/bin/lynx
(base) gsornsen@pop-os:~/git/rofi-web-search$ which st
/usr/local/bin/st
(base) gsornsen@pop-os:~/git/rofi-web-search$ which gnome-terminal
/usr/bin/gnome-terminal

```

**Comment by pdonadeo** [comment]
Thanks @gsornsen, I'll review the PR tonight, I'm at work now.

**Comment by pdonadeo** [comment]
Closed by c5c6db5
